### PR TITLE
protobuf -> 21.5

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=protobuf CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/protobuf.rb [hunk 1]
```diff
@@ -1,41 +1,47 @@
...
+# https://github.com/archlinux/svntogit-packages/raw/packages/protobuf/trunk/PKGBUILD
+
 require 'package'
 
 class Protobuf < Package
-  description 'Protocol buffers are a language-neutral, platform-neutral extensible mechanism for serializing structured data.'
+  description 'Protocol Buffers - Googles data interchange format'
   homepage 'https://developers.google.com/protocol-buffers/'
-  version '3.9.2'
+  @_ver = '21.5'
```
Comment (uberhacker) [inline packages/protobuf.rb:9]: Wow, that is quite a jump.

**Review by uberhacker — APPROVED** [review]

File: packages/protobuf.rb (hunk 1 above)
Comment (satmandu) [reply packages/protobuf.rb:9]: Right? They totally changed versioning...

**Review by supechicken — APPROVED** [review]

File: packages/protobuf.rb [hunk 2]
```diff
@@ -1,41 +1,47 @@
...
-           "--libdir=#{CREW_LIB_PREFIX}"
-    system 'make'
+    FileUtils.mkdir('builddir')
+    Dir.chdir('builddir') do
+      system "cmake #{CREW_CMAKE_OPTIONS} \
+      -Dprotobuf_BUILD_SHARED_LIBS=ON \
+      -Dprotobuf_BUILD_TESTS=OFF \
+      ../ -G Ninja"
+    end
+    system 'samu -C builddir'
```
Comment (supechicken) [inline packages/protobuf.rb:41]: Move `samu` into the `Dir.chdir` block?

Comment (supechicken) [reply packages/protobuf.rb:41]: (or use `chdir: 'builddir'` argument in `cmake` line)

Comment (satmandu) [reply packages/protobuf.rb:41]: Looking at usage in other packages:

```
/usr/local/lib/crew/packages $ grep "'samu -C builddir'" * | wc -l
34
```
So this is standard usage for many packages... why change it?

Comment (supechicken) [reply packages/protobuf.rb:41]: Now we have 2 `chdir` calls, one in `Dir.chdir` and one is done by `samu` (according by the `-C` argument)

My through is to reduce the `chdir` system call if possible (although that might not make it faster)

Comment (satmandu) [reply packages/protobuf.rb:41]: `samu` is already opening in a subshell. I suspect the increase in speed from removing an internal chdir call for samu is de minimus... and is that even a chdir? samu is merely being told which dir to start working from when it is invoked.

@uberhacker @saltedcoffii Thoughts?

Comment (uberhacker) [reply packages/protobuf.rb:41]: I think the code is fine the way it is now.

**Pull request merged**